src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.ts

```diff
@@ -45,15 +49,22 @@ export class DotThemeSelectorDropdownComponent implements OnInit, ControlValueAc
             .pipe(
                 pluck('identifier'),
                 mergeMap((identifier: string) => {
-                    this.currentSiteIdentifier = identifier;
                     this.paginatorService.setExtraParams('hostId', identifier);
                     return this.paginatorService.getWithOffset(0).pipe(take(1));
                 }),
                 take(1)
             )
             .subscribe((themes: DotTheme[]) => {
                 this.themes = themes;
```

Comment:
with this, we might have the same problem of the container selector, that the page do not refresh on pagination.